docs/guides/guide-for-latex-users.md

```diff
@@ -670,3 +670,22 @@ applicable, contains possible workarounds.
   break will be created if this spot would be a natural page break anyways. You
   can also use `[#v(1fr)]` to distribute space on your page. It works quite
   similar to LaTeX's `\vfill`.
+
+- **Separation of `import` and `include` behaviours.** `\include`ing a file in LaTeX
+  has two behaviours that have been separated in Typst: imports LaTeX macros,
+  and adds the document content to the parent file. In Typst: `#import`ing will
+  add definitions (equivalent to LaTeX macros) to the parent file,
+  but not add document content; `#include`ing will add document content
+  to the parent file, but will not import definitions.
+  The syntax for including document content is simply `#include "file.typ"`,
+  but the syntax for importing definitions provides more flexibility:
+  `#import "file.typ": *` will import all of `file.typ`'s definitions;
+  `#import "file.typ": x, y, template` will import only the named definitions.
+  It is of particular note for those familiar with LaTeX that
+  in LaTeX, a file only needs to be `\include`d once;
+  in Typst, your global definitions
+  and global setups need to be placed in a separate file,
+  and that file is then `#import`ed and/or `#include`d
+  into all of your project's nested files in addition to the main file.
```

Comment:
```suggestion
  into the files that make use of these definitions.
```

Just like with every good programming language, you only need to import stuff to where you use it.

Reply:
This would call for three boilerplate lines of code then.
1. Any `#include`s.
2. `#import` of definitions specific to the file.
3. `#include`/`#import` of setups that are used in every file, e.g. default font.